cross/ffmpeg/PLIST

```diff
@@ -1,134 +1,35 @@
-include/libavutil/sha512.h
-include/libavutil/sha.h
-include/libavutil/timecode.h
-include/libavutil/time.h
-include/libavutil/timestamp.h
-include/libavutil/version.h
-include/libavutil/xtea.h
-include/libswresample/swresample.h
-include/libswresample/version.h
-include/libswscale/swscale.h
-include/libswscale/version.h
-lib/libavcodec.a
-lib/libavdevice.a
-lib/libavfilter.a
-lib/libavformat.a
-lib/libavutil.a
-lib/libswresample.a
-lib/libswscale.a
-lib/pkgconfig/libavcodec.pc
-lib/pkgconfig/libavdevice.pc
-lib/pkgconfig/libavfilter.pc
-lib/pkgconfig/libavformat.pc
-lib/pkgconfig/libavutil.pc
-lib/pkgconfig/libswresample.pc
-lib/pkgconfig/libswscale.pc
-share/ffmpeg/examples/decoding_encoding.c
-share/ffmpeg/examples/demuxing.c
-share/ffmpeg/examples/filtering_audio.c
-share/ffmpeg/examples/filtering_video.c
-share/ffmpeg/examples/Makefile
-share/ffmpeg/examples/metadata.c
-share/ffmpeg/examples/muxing.c
-share/ffmpeg/examples/README
-share/ffmpeg/examples/resampling_audio.c
-share/ffmpeg/examples/scaling_video.c
-share/ffmpeg/ffprobe.xsd
-share/ffmpeg/libvpx-1080p50_60.ffpreset
-share/ffmpeg/libvpx-1080p.ffpreset
-share/ffmpeg/libvpx-360p.ffpreset
-share/ffmpeg/libvpx-720p50_60.ffpreset
-share/ffmpeg/libvpx-720p.ffpreset
-share/ffmpeg/libx264-ipod320.ffpreset
-share/ffmpeg/libx264-ipod640.ffpreset
-share/man/man1/ffmpeg.1
-share/man/man1/ffmpeg-all.1
-share/man/man1/ffmpeg-bitstream-filters.1
-share/man/man1/ffmpeg-codecs.1
-share/man/man1/ffmpeg-devices.1
-share/man/man1/ffmpeg-filters.1
-share/man/man1/ffmpeg-formats.1
-share/man/man1/ffmpeg-protocols.1
-share/man/man1/ffmpeg-resampler.1
-share/man/man1/ffmpeg-scaler.1
-share/man/man1/ffmpeg-utils.1
-share/man/man1/ffprobe.1
-share/man/man1/ffprobe-all.1
-share/man/man1/ffserver.1
-share/man/man1/ffserver-all.1
-share/man/man3/libavcodec.3
-share/man/man3/libavdevice.3
-share/man/man3/libavfilter.3
-share/man/man3/libavformat.3
-share/man/man3/libavutil.3
-share/man/man3/libswresample.3
-share/man/man3/libswscale.3
+bin:bin/ffmpeg
+bin:bin/ffprobe
+bin:bin/ffserver
+lnk:lib/libavcodec.so
+lnk:lib/libavcodec.so.57
+lib:lib/libavcodec.so.57.24.102
+lnk:lib/libavdevice.so
+lnk:lib/libavdevice.so.57
+lib:lib/libavdevice.so.57.0.101
+lnk:lib/libavfilter.so
+lnk:lib/libavfilter.so.6
+lib:lib/libavfilter.so.6.31.100
+lnk:lib/libavformat.so
+lnk:lib/libavformat.so.57
+lib:lib/libavformat.so.57.25.100
+lnk:lib/libavutil.so
+lnk:lib/libavutil.so.55
+lib:lib/libavutil.so.55.17.103
+lnk:lib/libpostproc.so
+lnk:lib/libpostproc.so.54
+lib:lib/libpostproc.so.54.0.100
+lnk:lib/libswresample.so
+lnk:lib/libswresample.so.2
+lib:lib/libswresample.so.2.0.101
+lnk:lib/libswscale.so
+lnk:lib/libswscale.so.4
+lib:lib/libswscale.so.4.0.100
+shr:share/ffmpeg/ffprobe.xsd
+shr:share/ffmpeg/libvpx-1080p50_60.ffpreset
+shr:share/ffmpeg/libvpx-1080p.ffpreset
+shr:share/ffmpeg/libvpx-360p.ffpreset
+shr:share/ffmpeg/libvpx-720p50_60.ffpreset
+shr:share/ffmpeg/libvpx-720p.ffpreset
+shr:share/ffmpeg/libx264-ipod320.ffpreset
```

Comment:
L34-L35 are no longer present according to the error's I'm seeing. 
Please recreate the PLIST to ensure no other files have been added/removed.